Assets/_Developers/Rolo/Scripts/MapRework/StationMapItem.cs

```diff
@@ -0,0 +1,14 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+public class StationMapItem
```

Comment:
The use of a class is useful if there are functions or it needs to be serialized. In the current state, this class might be better converted into a "struct".